ClassHierarchyAndCollections/ClassHierarchyAndCollectionsTests/BusinessTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using ClassHierarchyAndCollections;
+
+namespace ClassHierarchyAndCollectionsTests
+{
+    /// <summary>
+    /// Defines tests for the <see cref="Business"/> class.
+    /// </summary>
+    [TestClass]
+    public class BusinessTests
+    {
+        /// <summary>
+        /// This method tests to see if an instance of the "Business" class can be created.
+        /// </summary>
+        [TestMethod]
+        public void CreateInstanceBusinessClass()
+        {
+            Business sampleBusiness = new Business();
+            Assert.IsNotNull(sampleBusiness);
+        }
+
+        /// <summary>
+        /// This method checks that the "Business" class derives from the "Organization" Class
+        /// </summary>
+        [TestMethod]
+        public void DerivesCorrectlyBusinessClass()
```

Comment:
You could also include a test to confirm that Business implements the ILocatable interface using the Assert.IsInstanceOfType method.